test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -43,10 +43,8 @@ def test_tensorflow_train_mnist_gpu(tensorflow_training, ec2_connection, gpu_onl
     execute_ec2_training_test(ec2_connection, tensorflow_training, TF_MNIST_CMD)
 
 
-# TODO: Change this back TF_EC2_CPU_INSTANCE_TYPE. Currently this test times out on c4.8x, m4.16x and t2.2x,
-#       though passes on all three when run manually. For now we are pinning to c5.18 until we can resolve the issue.
 @pytest.mark.model("mnist")
-@pytest.mark.parametrize("ec2_instance_type", ["c5.18xlarge"], indirect=True)
+@pytest.mark.parametrize("ec2_instance_type", TF_EC2_CPU_INSTANCE_TYPE, indirect=True)
```

Comment:
May I know a context that why they are using "c5.18xlarge" as default originally?

Reply:
I believe the test was timing out on other instance types

Reply:
Yes, it was timing out for other instance types.